### DISCUSSION TITLE
The screen display has a  with the line bar. Is there anyway to fix this?

### DISCUSSION THREAD

**Discussion by kevinW0W** [rocksdanister/lively#2086] [Q&A]
![Screenshot 2024-01-22 154024](https://github.com/rocksdanister/lively/assets/152757721/38d892c3-608d-47a2-bcff-eb6f1fdc5ebf)
When i rotate my screen and back again it has a gray line bar thing.

**Comment by kevinW0W** [comment]
My device log
### Info system

- Device name	SAMSUNG-MASTER
- Processor	12th Gen Intel(R) Core(TM) i5-1235U   1.30 GHz
- Installed RAM	8.00 GB (7.70 GB usable)
- Device ID	########-####-####-####-############
- Product ID	#####-#####-#####-#####
- System type	64-bit operating system, x64-based processor
- Pen and touch	Pen and touch support with 10 touch points


### Windows
-  Edition	Windows 11 Home
- Version	23H2
- Installed on	‎1/‎11/‎2023
- OS build	22631.3007
- Experience	Windows Feature Experience Pack 1000.22681.1000.0

So you will know that what type using to fix an issues.

**Comment by kevinW0W** [comment]
> ![Screenshot 2024-01-22 154024](https://github.com/rocksdanister/lively/assets/152757721/38d892c3-608d-47a2-bcff-eb6f1fdc5ebf)
> 
> When i rotate my screen and back again it has a gray line bar thing.

By the way you might need to turn on light mode or white background order to see a black line.

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/issues/1996#issuecomment-1837313661

Does it reproduce with scaling at 100%?

**Reply by kevinW0W** [reply]
Yeah it is

**Reply by kevinW0W** [reply]
But the thing is the part where does it glitch is the Trangle & light. Cuz i tried to re scale to 100 but nothing changed when i rotate my laptop.

(Edited)
I have been reinstalling my app and it seems like it works.

**Reply by rocksdanister** [reply]
Does it happen with other html wallpaper like Rain?

**Reply by kevinW0W** [reply]
I don't think so cuz the only one issues is Triangles & light.

**Reply by rocksdanister** [reply]
Then it's an issue with the wallpaper code, if you are familiar with javascript programming can try to take a look.

**Reply by kevinW0W** [reply]
Ok

**Reply by kevinW0W** [reply]
Also can you fix the crash part cuz it suddenly crashed the live wallpaper at randomly.
